### ISSUE TITLE
Example from readme not working

### ISSUE THREAD

**Issue by AnHeuermann** [SciML/DiffEqOperators.jl#416]
I'm trying to run [example 1 from the readme](https://github.com/SciML/DiffEqOperators.jl#example-1-automated-finite-difference-solution-to-the-heat-equation) but am getting errors.

I had it working a few days ago, but hat do re-install all Julia packages because of something other and now can't get my previously running code to work again. I guess when reinstalling DiffEqOperators.jl a newer version was installed.

### How to reproduce
Copy the example into a `example.jl` and run it from REPL.
The resulting error message:

```repl
julia> include("example.jl")
ERROR: LoadError: type Int64 has no field val
Stacktrace:
 [1] getproperty(x::Int64, f::Symbol)
   @ Base .\Base.jl:33
 [2] (::DiffEqOperators.var"#190#252"{Int64})(d::Symbolics.VarDomainPairing)
   @ DiffEqOperators C:\Users\Username\.julia\packages\DiffEqOperators\FB7f2\src\MOLFiniteDifference\MOL_discretization.jl:51
 [3] findnext(testf::DiffEqOperators.var"#190#252"{Int64}, A::Vector{Symbolics.VarDomainPairing}, start::Int64)
   @ Base .\array.jl:1855
 [4] findfirst(testf::Function, A::Vector{Symbolics.VarDomainPairing})
   @ Base .\array.jl:1906
 [5] symbolic_discretize(pdesys::PDESystem, discretization::MOLFiniteDifference{Vector{Pair{Num, Float64}}, Int64})
   @ DiffEqOperators C:\Users\Username\.julia\packages\DiffEqOperators\FB7f2\src\MOLFiniteDifference\MOL_discretization.jl:51
 [6] discretize(pdesys::PDESystem, discretization::MOLFiniteDifference{Vector{Pair{Num, Float64}}, Int64})
   @ DiffEqOperators C:\Users\Username\.julia\packages\DiffEqOperators\FB7f2\src\MOLFiniteDifference\MOL_discretization.jl:369
 [7] top-level scope
   @ D:\some\path\example.jl:28
 [8] include(fname::String)
   @ Base.MainInclude .\client.jl:444
 [9] top-level scope
   @ REPL[1]:1
in expression starting at D:\some\path\example.jl:28
```

### Related issues
The readme was recently updated. Maybe I'm just not on the correct version of DiffEqOperators?
https://github.com/SciML/DiffEqOperators.jl/issues/401

### OS, Julia and packages
OS: Windwos 10
Julia Version: 1.6.1 (2021-04-23)
Installed Packages
```repl
  [6e4b80f9] BenchmarkTools v1.1.0
  [9fdde737] DiffEqOperators v4.29.0
  [0c46a032] DifferentialEquations v6.17.1
  [5b8099bc] DomainSets v0.5.2
  [961ee093] ModelingToolkit v5.21.0
  [1dea7af3] OrdinaryDiffEq v5.59.2
  [91a5bcdd] Plots v1.16.7
  [d330b81b] PyPlot v2.9.0
  [93e0c654] Reduce v1.2.11
  [37e2e46d] LinearAlgebra
```

**Comment by AnHeuermann** [comment]
After fiddling around some more with the example and comparing to https://github.com/SciML/DiffEqOperators.jl/blob/master/test/MOL/MOL_1D_Linear_Diffusion.jl I found the problem. It seems that two lines need to be updated.

Broken:
```julia
# PDE system
pdesys = PDESystem(eq,bcs,domains,[t,x],[u])

# Method of lines discretization
dx = 0.1
order = 2
discretization = MOLFiniteDifference([x=>dx],order)
```

Working:
```julia
# PDE system
pdesys = PDESystem(eq,bcs,domains,[t,x],[u(t,x)])

# Method of lines discretization
dx = 0.1
order = 2
discretization = MOLFiniteDifference([x=>dx],t;centered_order=order)
```

I'll create a PR to fix this, so I don't do the same mistake next week, again.

**Comment by ChrisRackauckas** [comment]
Thanks!

**Comment by AnHeuermann** [comment]
Fixed in d9438afa9c978817ab1aa2af54f6b6c01f6d67d1
